src/top/dashboard.rs

```diff
@@ -16,14 +18,46 @@ use tui::{
     Frame, Terminal,
 };
 
+/// Format metrics, with thousands separation
+trait ThousandsFormatter {
+    fn thousands_format(&self) -> String;
+}
+
+impl ThousandsFormatter for i64 {
+    fn thousands_format(&self) -> String {
+        match self {
+            0 => "--".into(),
+            _ => self.to_formatted_string(&Locale::en),
+        }
+    }
+}
+
+/// Format metrics, using the 'humanized' format, abbreviating with suffixes
+trait HumanFormatter {
```

Comment:
Not a big deal given the simplicity of this, but might be nice to throw a couple tests in.

Reply:
Thanks for prompting for tests, @jszwedko 👍

This is a good reminder why even seemingly obvious assumptions ought to be tested. Parsing a 13 digit `f64` panicked, which would have caught anyone with a `processed_bytes` value of > 1TB.

I swapped human-format out for [number-prefix](https://crates.io/crates/number_prefix), which also appears far more active (both downloads + updates.), and also added a `human_format_bytes` after realising that I'd neglected to accommodate binary stats with a 1024 multiple.

The new panel looks like this:

<img width="1326" alt="image" src="https://user-images.githubusercontent.com/719722/98859556-48162700-245a-11eb-957b-1f416167010c.png">